package.json

```diff
@@ -14,6 +14,7 @@
     "mapbox-gl": "^0.53.1",
     "material-ui-icons": "^1.0.0-beta.36",
     "mdi-react": "^5.3.0",
+    "prop-types": "^15.7.2",
```

Comment:
What's this for?

Reply:
This package is being used in line 83. It is used for run time checking of the props. It was in the initial commit and so I had not touched that portion but installed the package as intellij was throwing warning.